src/main/java/com/nike/cerberus/service/KmsPolicyService.java

```diff
@@ -112,7 +112,7 @@ protected boolean consumerPrincipalIsAnArnAndNotAnId(String policyJson) {
                             StringUtils.equals(statement.getId(), CERBERUS_CONSUMER_SID) &&
                                     statement.getPrincipals()
                                             .stream()
-                                            .anyMatch(principal -> awsIamRoleArnParser.isRoleArn(principal.getId())));
+                                            .anyMatch(principal -> awsIamRoleArnParser.isArnThatCanGoInKeyPolicy(principal.getId())));
```

Comment:
There was a bug here where if anything other than a role ARN was in a policy we considered it invalid.  Really we just want to catch IDs so we want to allow any ARN type that can go in a policy.